### ISSUE TITLE
documentation should include more examples for LIGO researchers

### ISSUE THREAD

**Issue by jrollins** [gwpy/gwpy#856]
The documentation for gwpy is one of it's strong suits.  However, it has recently become very focused on stuff useful for the public (such as access of open data), and less so for people working inside of LIGO.  Today I was trying to find examples of how to use gwpy for a new student and all the examples I could fine were about accessing open data.  All of the example commands for fetching data are now "fetch_open_data".

It would be great if there was a page equivalent to this one, but specifically for LIGO people:

https://gwpy.github.io/docs/stable/examples/timeseries/public.html

Thanks for the great and useful tool.

**Comment by duncanmmacleod** [comment]
@jrollins, I agree. There are restrictions on what proprietary data can be shown in such examples, but I would be delighted to here some suggestions of specific examples that would be of use.

**Comment by duncanmmacleod** [comment]
Migrated to https://gitlab.com/gwpy/gwpy/-/issues/856, please join us on gitlab to continue your contribution!
